### DISCUSSION TITLE
Following your guide, I never get the `code` in the search params

### DISCUSSION THREAD

**Discussion by Nedimko123** [supabase/supabase#16896] [Questions]
Following this guide: https://supabase.com/docs/guides/auth/auth-password-reset#server-side-rendering-ssr

I never get the `code` in the url search params it gets to be null for SSR auth

The url when I click on the email and it redirects me: real values replaced by bb:

My url `http://127.0.0.1:5173/auth/callback?next=/account/update-password#access_token=bb&expires_at=bb&expires_in=bb&refresh_token=bb&token_type=bearer&type=recovery`

**Comment by GaryAustin1** [comment]
![image](https://github.com/supabase/supabase/assets/54564956/b19d4e95-3d30-42e1-8fec-90c3aab4de6a)
Did you turn on pkce flow?

**Comment by Nedimko123** [comment]
I cannot do that with supabase-auth-helpers-qwik

**Reply by GaryAustin1** [reply]
You did not mention you were using auth-helpers.  You might ask in the repository for that.  I don't use it and am not up to speed on each of the versions and their support of pkce.

**Reply by J0** [reply]
Hey @Nedimko123,

Thanks for query - if it's still relevant, do you mind sharing the auth-helpers version you're using?
